types/algoliasearch/index.d.ts

```diff
@@ -1519,7 +1519,7 @@ declare namespace algoliasearch {
   }
 
   interface Task {
-    taskID: number;
+    taskID: Record<string, number>;
```

Comment:
Not all `Task`s are equal. Some do need to type `number` (https://www.algolia.com/doc/api-reference/api-methods/clear-objects/?language=javascript#json-format).
There needs to be separate interfaces for every different Task to avoid confusion

Reply:
I will update accordingly.